src/amo/reducers/reviews.js

```diff
@@ -176,6 +186,21 @@ export const changeViewState = ({
   };
 };
 
+export const getReviewsByUserId = (
+  state: ReviewState,
+  userId: number,
+): {|
+  reviewCount: number,
+  reviews: Array<UserReviewType>,
+|} | null => {
+  return state.byUserId[userId]
+    ? {
+        reviewCount: state.byUserId[userId].reviewCount,
+        reviews: state.byUserId[userId].reviews.map((id) => state.byId[id]),
```

Comment:
This should be `expandReviewObjects({ state, reviews: state.byUserId[userId].reviews })`